packages/adt-mcp/README.md

```
@@ -256,18 +256,39 @@ and a typed diagnostic when exact isolation cannot be proven.
-**Returns:** Plain ABAP source text.
+**Returns:**
+
+- Default: `{ object, bytes, source, version, include }`
+- `method=*`: `{ object, methodCount, methods }`
+- `method=<name>`: `{ object, method, startLine, endLine, bytes, source }`
+- `grep`: `{ object, pattern, matchCount, matches, methodContext }`
+- `format=structured`: `{ object, source, bytes, version, include, includes, methods }`
+
+**Examples:**
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)